### PR TITLE
Document how to share folders with the containers

### PR DESCRIPTION
**Pull request by pvdrz**
cc @dajamante @hoverbear

### CODE REVIEW

File: ferrocene/doc/internal-procedures/src/working-with-the-ci.rst [hunk 1]
```diff
@@ -188,6 +183,39 @@ of the job:
...
+can authenticate in the container run:
+
+.. code-block:: bash
+
+   docker run --rm --tty --interactive --workdir /ferrocene \
+      --mount "type=bind,src=$(pwd),dst=/ferrocene" \
+      --mount "type=bind,src=$HOME/.aws,dst=/home/ci/aws" \
+      ubuntu-20 bash
+
+Remember that any folders mounted with the `bind` type must exist beforehand.
```
Comment (tshepang) [inline ferrocene/doc/internal-procedures/src/working-with-the-ci.rst:218]: should clarify if the folders should exist on host or on guest

File: ferrocene/doc/internal-procedures/src/working-with-the-ci.rst [hunk 2]
```diff
@@ -188,6 +183,39 @@ of the job:
...
+folder for ``/ferrocene/build``:
+
+.. code-block:: bash
+
+   docker run --rm --tty --interactive --workdir /ferrocene \
+      --mount "type=bind,src=$(pwd),dst=/ferrocene" \
+      --mount "type=bind,src=$(pwd)/container-build,dst=/ferrocene/build" \
+      ubuntu-20 bash
+
+Note that you will need to create the `container-build` folder beforehand.
```
Comment (tshepang) [inline ferrocene/doc/internal-procedures/src/working-with-the-ci.rst:200]: should specify how to make build system use container-build/ instead of build/

**Comment by tshepang** [comment]
- since the second commit is a fixup, I would make it part of the first
- for consistency, I would use *directory* instead of *folder*

File: ferrocene/doc/internal-procedures/src/working-with-the-ci.rst (hunk 2 above)
Comment (pvdrz) [reply ferrocene/doc/internal-procedures/src/working-with-the-ci.rst:200]: you don't need to do anything for that, the container will see `container-build` as `build`.

File: ferrocene/doc/internal-procedures/src/working-with-the-ci.rst (hunk 1 above)
Comment (pvdrz) [reply ferrocene/doc/internal-procedures/src/working-with-the-ci.rst]: done

File: ferrocene/doc/internal-procedures/src/working-with-the-ci.rst (hunk 2 above)
Comment (tshepang) [reply ferrocene/doc/internal-procedures/src/working-with-the-ci.rst]: oh, indeed!

**Review by tshepang — APPROVED** [review]

**Comment by tshepang** [comment]
bors merge

**Pull request merged**